components/x-live-blog-post/readme.md

```diff
@@ -1,14 +1,14 @@
-# x-content-block
+# x-live-blog-post
 
-This module displays a block of content with an id, title, body and timestamp. Main use case is the new version of live blogs, although we don't want to limit its usage to it.  
+This module displays a live blog post with an id, title, body and timestamp.  
```

Comment:
lol thanks to Simon we now know time stamp is two words 😄 so just changing this to `time stamp` from `timestamp`